### ISSUE TITLE
Was the UnifiedQA-v2 baseline pre-trained on data after mid-2021 (testing questions range)?

### ISSUE THREAD

**Issue by qiyan98** [andyzoujm/autocast#8]
Hi,

Thanks for sharing your code and curating the datasets. I have a question about the potential information leakage of the UnifiedQA-v2 backbone, which was [released on Feb 2022](https://arxiv.org/pdf/2202.12359.pdf). But the Autocast testing set covers questions that resolve after mid-2021, according to the paper:
> Thus, we split our questions using a date cut-off of mid-2021, which means that questions in the test set resolve from mid-2021 to mid-2022.

I wonder if UnifiedQA-v2 uses any data after mid-2021 and has any information leakage issue. Indedd, the paper claims quite often about the importance of information leakage regarding predicting future with past data in a faithful and realistic way.
> Note that if a model is pre-trained on data from after mid-2021, this will also not simulate forecasting faithfully.

> Note that the T5 backbone for the baselines was pre-trained on data from before 2020, far before the timeline of the question, so retrieval provides vital information that models would not otherwise have.

Thanks,
Qi

**Comment by andyzoujm** [comment]
Hi, UnifiedQA-v2 was trained on numerous downstream tasks, some of which may contain information after mid-2021. To follow the standard practice, one might want to filter out any datasets that were curated after mid-2021 from UnifiedQA's training data and reproduce the model with only the remaining datasets.
